builds/azure-pipelines/build-pr.yml

```diff
@@ -22,10 +22,11 @@ stages:
       matrix:
         linux:
           imageName: 'ubuntu-latest'
-        mac:
-          imageName: 'macos-latest'
-        windows:
-          imageName: 'windows-latest'
+        # Disabling other platform builds temporarily until we can get integration tests working there
```

Comment:
I would rather we do something else for the platforms that don't support integration tests currently such that the unit tests still run. I'd suggest looking into the filter option to dotnet test - adding a category seems like a decent idea (Categories being Unit/Integration)

Reply:
Added filter to exclude integration tests on Mac and Windows